app/policies/resource_policy.rb

```diff
@@ -9,4 +9,8 @@ def initialize(user, resource)
   def show?
     true
   end
+
+  def update?
+    true
```

Comment:
But the owner is optional for resources (we mostly have orphans resources actually).